src/gui/newwizard/setupwizardcontroller.cpp

```diff
@@ -33,6 +33,8 @@ QString initLocalFolder()
     return OCC::FolderMan::instance()->findGoodPathForNewSyncFolder(localFolder);
 }
 
+const QString defaultUrlScheme = QStringLiteral("https://");
```

Comment:
I'd move constants to the top of the anonymous namespace.
Historically the constants at least for strings where followed the schema `fooC`.
The C might have meant CString or constant, I don't care but we should come up with a standard for constants.

I'd suggest to either keep the C or to start Uppercase?